src/ApiPort/ApiPort.VisualStudio/Views/AnalysisOutputToolWindowControl.xaml.cs

```diff
@@ -8,16 +8,12 @@ namespace ApiPortVS
 {
     public partial class AnalysisOutputToolWindowControl : UserControl
     {
-        public static OutputViewModel Model { get; } = new OutputViewModel();
```

Comment:
I removed this and put it into our [container builder registration code](https://github.com/Microsoft/dotnet-apiport/pull/593/files#diff-7ebc0d92afa75b565fba2f0c43caea89R98) so that we didn't have this random static lying around.